core/src/main/kotlin/maru/consensus/ForkId.kt

```diff
@@ -8,64 +8,48 @@
  */
 package maru.consensus
 
-import maru.core.Hasher
-import maru.database.BeaconChain
-import maru.serialization.Serializer
+import java.nio.ByteBuffer
+import java.util.zip.CRC32
+import org.apache.tuweni.bytes.Bytes
 
 data class ForkId(
   val chainId: UInt,
-  val forkSpec: ForkSpec,
   val genesisRootHash: ByteArray,
 ) {
+  companion object {
+    const val FORK_ID_FIELD_NAME = "mfid"
```

Comment:
can just move this to the top of the `ForkId.kt` file outside of the data class to avoid the need for a companion object